README.md

```diff
@@ -1,45 +1,117 @@
+
+* **Role-Based Routing**
+We also added in role-based routing. Because some actions should not be exposed to regular users (students) browsing the site, such as deleting or adding courses, we decided to add separate roles for administrators and generic users. Based on the role a user logs in with, they will be limited to only the necessary actions for their use case.
+
+---
+**Next Steps:**
 
-Project Task Requirements
-Minimal Requirements:
-* Front end page to view list of courses
-* Back end that takes user input and stores data
-* Database to store course reviews added by users
+Our next steps are to add User Profile pages to CourseHub, enabling users to edit their profile to provide their education statuses and keep track of their comment histories. We would also like to add additional features such as giving users the ability to up and down vote reviews, and being able to display a summary for each course of frequently-mentioned words in user reviews.
 
-Standard Requirements:
-* Front end page for each individual course
-* Add ratings/summary for each course
-* Search function using keywords
-* Filter for courses(i.e. four stars and above)
-* Contact form/Button to report issues
+---
+**Team Contributions:**
 
-Stretch Requirements:
-* Add historical data for each course
-* User Academic Profile (i.e. Computer Science 2nd Year)
+Aaron:
+* Set up CD pipline
+* Implemented and finalized Facebook and Google authentication login functionality
+* Created Login component and related functionality
+* Implemented 'Add Review' and 'Add Rating' functionality
+* Added pages to homepage
+* Set up backend search and filter
+* Set up environment variables
+* Created CourseItem and CourseList components
+* Added Material-UI to components
+* Created initial UI design using Figma
+* Continuous bug fixes and refactoring
 
+Winnie:
+* Set up Express server, models, and API endpoints
+* Managed and set up database with usable dummy data
+* Managed back end functionality
+* Restructured repository and set up initial deployment
+* Added 'Add Course' functionality
+* Created CourseOverview component
+* Set up redux-thunk for async fetch for courses and reviews
 
-Task Breakdown
-Front end page to view list of course:
-* User input to add course reviews
-* Add button to add reviews
-* List view of courses and it’s reviews
-* Dummy Data for courses
+Illean:
+* Set up About and Contact pages
+* Set up React routing for all components
+* Implemented front end search and filter functionality
+* Created ReviewItem and ReviewList components
+* Applied UI to components from Figma
+* Implemented 'Delete Course' functionality
 
-Relevant Links
+Spandana:
+* Set up Google login and linked authentication to back end
+* Implemented 'Delete All Courses' functionality (removed later)
+* Created Header and Button component
+* Final code cleanup
+* Investigated and implemented role-based user routing
```

Comment:
Minor nit: Will also be working on handling signup

Reply:
yes i will add! thanks